### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by chukrello**

### CODE REVIEW

File: prog.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
+text = open('INPUT.TXT').read()
+result = 'YES'
+letters=['A','B','C','D','E','F','G','H']
+ 
+def check_size():
```
Comment (chukrello) [inline prog.py:5]: Проще сделать
`return len(size) == 5`

File: prog.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    else:
+        return False
+   
+def numbers_1(a, b):
+    if abs(a - b) == 1:
+        return True
+    else:
+        return False
+ 
+size = check_size()
```
Comment (chukrello) [inline prog.py:48]: неправильно названа переменная

File: prog.py [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+result = 'YES'
+letters=['A','B','C','D','E','F','G','H']
+ 
+def check_size():
+    if len(text) == 5:
+        return True
+    else:
+        return False
+ 
+def check_defis():
```
Comment (svetomsk) [inline prog.py:11]: PEP 8 requires two lines between functions

File: prog.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+        return False
+ 
+def check_alphabet (a):
+    s = False
+    for i in letters:
+        if a == i:
+            s += True
+        else:
+            s += False
+    if s == True:
```
Comment (svetomsk) [inline prog.py:24]: Can be simplified to just _return s_

File: prog.py [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+        return True
+    else:
+        return False
+       
+def check_numbers(n):
```
Comment (svetomsk) [inline prog.py:29]: We can use conditions like 1 <= n <= 8:

File: prog.py [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+        return True
+    else:
+        return False
+ 
+def diapazon_numbers(a, b):
+    r = abs(a - b)
+    if r >= 1 and r <= 2:
+        return True
+    else:
+        return False
```
Comment (svetomsk) [inline prog.py:40]: Just return 1 <= r <= 2

File: prog.py [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+        else:
+            result = 'NO'
+    else:
+        result = 'NO'
+else:
+    result = 'ERROR'
+   
+file = open('OUTPUT.TXT', 'w')
+file.write(str(result))
+file.close()
```
Comment (svetomsk) [inline prog.py:79]: Why don't you split this part into functions or at least add some comments?

**Review by svetomsk — CHANGES_REQUESTED** [review]

**Pull request closed without merging**